js/main.js

```diff
@@ -1,2 +1,26 @@
 /* global data */
 /* exported data */
+var $entryForm = document.querySelector('.entry-form');
+var $imageInput = document.querySelector('#image-url');
+var $entryImage = document.querySelector('.entry-image');
+
+function handleImageUrlInput(event) {
+  $entryImage.setAttribute('src', event.target.value);
+}
+
+function handleEntrySubmit(event) {
+  event.preventDefault();
+  var entry = {
+    image: $entryForm.elements.image.value,
+    title: $entryForm.elements.title.value,
+    notes: $entryForm.elements.notes.value,
+    entryId: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.push(entry);
```

Comment:
This should be being prepended to the array, not appended to the end. You will need a different method than `push` here 🤓